### PR TITLE
Make CI use virtualenv

### PR DESCRIPTION
**Pull request by kevin1055**

### CODE REVIEW

File: launch [hunk 1]
```diff
@@ -1,4 +1,8 @@
 #!/bin/sh -x
 kill $(cat lendbot.pid)
+pip install virtualenv
+virtualenv venv
+source venv/bin/activate
```
Comment (blerud) [inline launch]: how does source interact w/ running in bash file

maybe just use venv/bin/pip or w/e

File: launch [hunk 2]
```diff
@@ -1,4 +1,8 @@
 #!/bin/sh -x
 kill $(cat lendbot.pid)
+pip install virtualenv
```
Comment (blerud) [inline launch]: can prob assume we have virtualenv isntalled

File: launch (hunk 1 above)
Comment (kevin1055) [reply launch]: https://superuser.com/questions/46139/what-does-source-do

File: launch (hunk 2 above)
Comment (kevin1055) [reply launch]: but u don't

File: launch (hunk 1 above)
Comment (blerud) [reply launch]: fk u

File: launch (hunk 2 above)
Comment (blerud) [reply launch]: yea we do

Comment (kevin1055) [reply launch]: ok edit it

**Pull request merged**